github-release/main.go

```diff
@@ -0,0 +1,189 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"os"
+	"os/signal"
+	"strings"
+	"syscall"
+	"time"
+
+	"github.com/google/go-github/v50/github"
+	"github.com/grafana/grafana-github-actions-go/pkg/changelog"
+	"github.com/grafana/grafana-github-actions-go/pkg/ghgql"
+	"github.com/grafana/grafana-github-actions-go/pkg/toolkit"
+	"github.com/rs/zerolog"
+	"github.com/spf13/pflag"
+)
+
+var makeLatestTrue = "true"
+var makeLatestFalse = "false"
+
+func main() {
+	logger := zerolog.New(zerolog.ConsoleWriter{Out: os.Stderr}).With().Timestamp().Logger()
```

Comment:
what's the value of zerolog? This isn't a high performance application; I think I'd rather we just use the stdlib logger until we need something else, no?